packages/snaps-utils/src/handlers.ts

```diff
@@ -96,6 +103,7 @@ export type OnRpcRequestHandler<Params extends JsonRpcParams = JsonRpcParams> =
  */
 export type OnTransactionResponse = {
   content: Component | null;
+  severity?: SeverityLevel;
```

Comment:
Should be wrapped in `EnumToUnion` (so you can use both `'critical'` and `SeverityLevel.Critical`).